tutorials/remote_scan_tutorial/RemoteScanTutorial.ipynb

```diff
@@ -0,0 +1,968 @@
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Saved template to: ./scan_definition.yaml\n"
+     ]
+    }
+   ],
+   "source": [
+    "scan_file=\"./scan_definition.yaml\"\n",
+    "with open(scan_file, \"w\") as f:\n",
+    "    scan.save(f)\n",
+    "    print(f\"Saved template to: {scan_file}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step (4) Use Certifai Remote Scan for this saved definition file\n",
+    "**Task 1) Create a remote config using the kubeconfig.json file and the alias**\n",
+    "\n",
+    "**The two things that need to be changed are:**\n",
+    "- *file*: In order to run remote scan jobs use the certifai-kubeconfig.json file that is downloaded during the Certifai Pro/Enterprise setup.\n",
+    "- *remote_alias*: Alias name to retrieve the Certifai remote config"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {
+    "gather": {
+     "logged": 1628246584611
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Checking for access to Kubernetes cluster with context - certifai-pro\n",
+      "Connection to cluster succeeded, found API - v1\n",
+      "Updating alias - gke\n",
+      "Configuration updated from - ~/.kube/config\n"
+     ]
+    }
+   ],
+   "source": [
+    "from certifai.client.remote import remote_config, remote_list\n",
+    "\n",
+    "class Options:\n",
+    "    def __init__(self, **kwargs):\n",
+    "        self.__dict__.update(kwargs)\n",
+    "\n",
+    "remote_alias = '' #Add name for the alias\n",
+    "pro_kubeconf_locn = '' # Enter the path for'certifai-kubeconfig.json'\n",
+    "\n",
+    "args = Options(file=pro_kubeconf_locn,  \n",
+    "                alias=remote_alias,\n",
+    "                context='current-context',\n",
+    "                namespace='certifai',\n",
+    "                timeout=15)\n",
+    "\n",
+    "r_conf = remote_config.remote_config(args)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Task 2) Run a remote scan using the scan definition file that we have created**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {
+    "gather": {
+     "logged": 1628246587187
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Created certifai scan ID aa15ea4f59c8\n"
+     ]
+    }
+   ],
+   "source": [
+    "from certifai.client.remote.remote_scan import remote_scan\n",
+    "scan_file = \"scan_definition.yaml\"\n",
+    "list_args = Options(alias=remote_alias, definition_file=scan_file, config=[], cpu_req=None, details=False, dry_run=False, info=False, mem_req=None, model=None, model_id_tags=None, output=None, parallel=0, preflight=False, report=None)\n",
+    "remote_scan(parsed_args=list_args, args=[])"
```

Comment:
The `args` parameter should be the list equivalent to `parsed_args` - this is an example of how these functions are closely tied with the CLI. The argument list is an attribute of the scan job, and the operator expects to parse those strings to create the container args.

So, if you set a value in the `Options` instance and don't set the equivalent in `args` list **it won't be reflected in the remote job**.